src/components/facet-browser/helper.ts

```diff
@@ -100,4 +100,41 @@ export function getAllFilterPathsAsString(filterObject: {
   return allFilterPathsAsString;
 }
 
+const handleFacetFieldTranslationError = (name: string): never => {
+  throw new Error(`Didn't expect "${name}" in getFacetFieldTranslation`);
+};
+
+export const getFacetFieldTranslation = (name: string): string => {
```

Comment:
As far as I can tell this addition needs some more work and should not be merged as is.